src/spaces/lchuv.js

```diff
@@ -0,0 +1,68 @@
+import ColorSpace from "../space.js";
+import Luv from "./luv.js";
+import {constrain as constrainAngle} from "../angles.js";
+
+export default new ColorSpace({
+	id: "lchuv",
+	name: "LChuv",
+	coords: {
+		l: {
+			refRange: [0, 100],
+			name: "Lightness"
+		},
+		c: {
```

Comment:
@LeaVerou  LCH<sub>uv</sub>, unlike LCH<sub>ab</sub> has a saturation as well. Saturation is chroma / lightness. How should that be exposed, as a utility method? Or can `lchuv.s` be made to work somehow?

Reply:
The MVP won't have it, and you (or someone else) needs to open a new issue to discuss how to handle this in the API. 🥲